frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -42,7 +42,11 @@ export type TopologyDisplayOption = {
 export type DisplayFilters = TopologyDisplayOption[];
 
 // Applies the filters on the model and returns the ids of filters that were relevant
-export type TopologyApplyDisplayOptions = (model: Model, filters: DisplayFilters) => string[];
+export type TopologyApplyDisplayOptions = (
+  model: Model,
+  filters: DisplayFilters,
+  groupsExpanded?: boolean,
```

Comment:
Why isn't `groupsExpanded` just another type of display filter?
I was thinking that any display options could be handled the same way.

Then hide the usage behind a single hook which can fetch the expand display filter and only if the expand filter is `true` does it fetch the individual expand filter.
```
const isExpanded = useExpand(MY_DISPLAYFILTER);
```